samples/skills/QASkill/MemoryQuery/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "schema": 1,
+  "description": "",
+  "type": "completion",
+  "completion": {
+    "max_tokens": 1024,
+    "temperature": 0.8,
+    "top_p": 0.0,
+    "presence_penalty": 0.0,
+    "frequency_penalty": 0.0,
+    "stop_sequences": [
+      "[done]"
```

Comment:
the prompt doesn't mention to end the answer with this token. Either we update the prompt, or we can leave the list empty.

Reply:
This has been resolved.